### PR TITLE
Specify entries to be added to each switch

### PR DESCRIPTION
**Pull request by theojepsen**
- add custom entries to each switch
- dump PCAPs and persist them after container is stopped
- save debug log from each switch

The new sections in the README describe these changes in more detail.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -230,13 +230,44 @@ the multiswitch target in the manifest. For example, have a look at the
 [manifest](examples/multiswitch.p4app/p4app.json) for the multiswitch example
 app.
 
+
+#### Specifying entries for each switch
+The `ipv4_lpm`, `send_frame` and `forward` tables are automatically populated
+with this target. Additionally, you can specify entries to be added to each
+switch. You can either include a commands file, or an array of entries. For
```
Comment (cc10512) [inline README.md]: What happens if entries overlap between the file and the specific entries? Which one has precedence?

**Comment by sethfowler** [comment]
I had the same idea about supporting either a filename or inline text for situations like this - great minds think alike. =)

This looks good to me. Once you address the issue @cc10512 pointed out, this should be ready to merge.

File: README.md (hunk 1 above)
Comment (theojepsen) [inline README.md]: Good point. Custom entries have precedence now. I've also updated the readme to explain this.

**Comment by sethfowler** [comment]
Thanks, Theo!

**Pull request merged**